src/main/java/com/google/firebase/auth/ProviderConfig.java

```diff
@@ -60,6 +62,24 @@ static void assertValidUrl(String url) throws IllegalArgumentException {
     }
   }
 
+  static List<Object> getNestedList(Map<String, Object> outerMap, String id) {
```

Comment:
Given these are only used in `SamlProviderConfig`, how about pushing them down to the child class?

Reply:
Done.